docs/security/docs/020-update-tls-certificates.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Update TLS certificate
+type: Details
+---
+
+The TLS certificate is a vital security element. This document describes how to update the TLS certificate in Kyma.
+
+>**NOTE:** This procedure can interrupt the communication between your cluster and the outside world for a limited 
```

Comment:
How can we avoid that? What about rolling update?

Reply:
Rolling update should do the trick, but we have only one instance of ingress gateway by default. We would need to increase size of ingress gateway replica set.  It won't work on minikube as we're using hostPort.

Reply:
Rolling update should do the trick, but we have only one instance of ingress gateway by default. We would need to increase size of ingress gateway replica set.  It won't work on minikube as we're using hostPort.

Also istio will be adding hot-reloading of certificates in Q4, as here: https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!msg/istio-users/dyBMP0OlMBY/70MTj8kzBwAJ.

Reply:
Rolling update should do the trick, but we have only one instance of ingress gateway by default. We would need to increase size of ingress gateway replica set.  It won't work on minikube as we're using hostPort.

Also istio will be adding hot-reloading of certificates in Q4, as pointed here: https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!msg/istio-users/dyBMP0OlMBY/70MTj8kzBwAJ.

Reply:
Rolling update should do the trick, but we have only one instance of ingress gateway by default. We would need to increase size of ingress gateway replica set.  It won't work on minikube as we're using hostPort.

Also istio will be adding hot-reloading of certificates in Q4, as pointed here: https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!msg/istio-users/dyBMP0OlMBY/70MTj8kzBwAJ.

Reply:
Follow up: #1372

Reply:
Follow up: #1372

Reply:
Follow up: #1372